src/panels/config/automation/trigger/types/ha-automation-trigger-mqtt.ts

```diff
@@ -1,12 +1,15 @@
-import "@polymer/paper-input/paper-input";
 import { html, LitElement } from "lit";
 import { customElement, property } from "lit/decorators";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+import { HaFormSchema } from "../../../../../components/ha-form/types";
 import { MqttTrigger } from "../../../../../data/automation";
 import { HomeAssistant } from "../../../../../types";
-import {
-  handleChangeEvent,
-  TriggerElement,
-} from "../ha-automation-trigger-row";
+import { TriggerElement } from "../ha-automation-trigger-row";
+
+const SCHEMA: HaFormSchema[] = [
+  { name: "topic", selector: { text: {} } },
+  { name: "payload", required: false, selector: { text: {} } },
```

Comment:
required: false is the default

Reply:
required is default true

because thats what it was before I updated the text fields to even have the ability to be optional